Scheduler/main.c

```diff
@@ -12,6 +13,7 @@ void PrintScheduleData(ScheduleData* inputData)
     {
         case FCFS:
             printf("Scheduler Type: First-Come First-Served\n");
+	    RunProcess(inputData);
```

Comment:
Wrong place for your call to `RunProcess`. This should be in the switch statement in main, right under the comment "Call First-Come First-Served code". It will never execute except when you build with `PRINT_PROCESSES_DATA` set to true.

Reply:
Fixed in next commit